lib/volcanic/authenticator/v1/scope.rb

```diff
@@ -44,10 +47,37 @@ def to_s
         base
       end
 
+      def <=>(other)
+        score < other.score
+      end
+
+      # For each element in the VRN, assign it a value
+      # element = * (0)
+      # element is value (0.5)
+      # Stack multiply 1
+      # Dataset multiply 2
+      # Resource multiply 3
+      # ResourceId multiply 4
+      # Qualifiers multiply 5
+      def score
```

Comment:
can this be renamed? I fear that a method named "score" is likely to be confused....

consider `specificity_score`